hugegraph-python-client/src/pyhugegraph/utils/huge_config.py

```diff
@@ -45,7 +44,7 @@ def __post_init__(self):
         else:
             try:
                 response = requests.get(
-                    f"http://{self.ip}:{self.port}/versions", timeout=0.5
+                    f"{self.url}/versions", timeout=0.5
```

Comment:
Perhaps we should add a compatibility check/adapter here. 

If str does not start with `http/https`, it will automatically concatenate `http://` to avoid making modifications at all original calls. 🤔